src/Util/ParamsIterator.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * BigBlueButton open source conferencing system - https://www.bigbluebutton.org/.
+ *
+ * Copyright (c) 2016-2023 BigBlueButton Inc. and by respective authors (see below).
+ *
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation; either version 3.0 of the License, or (at your option) any later
+ * version.
+ *
+ * BigBlueButton is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace BigBlueButton\Util;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @internal
+ *
+ * @coversNothing
+ */
+class ParamsIterator extends TestCase
```

Comment:
🎨 : I think, as this is a util for our tests only, we should provide this as a trait and move it to the `tests` folder, so it is not delivered in the production code, since consumer cannot use it anyway due to being `@internal`.

Reply:
And: That class/trait should be named more specific to express the exact purpose.

Reply:
I removed the class/trait and added it to our other assert methods in the TestCase class